File: src/couchapps/WMStats/views/latestRequest/reduce.js
function(keys, values, rereduce) {
    var latestRequest = values[0];
    for (var i = 1; i < values.length; i++) {
        if (latestRequest.timestamp < values[i].timestamp) {latestRequest = values[i]};
    }
    return latestRequest;
}

File: src/couchapps/WMStatsErl/views/latestRequest/reduce.erl
fun(Keys, Values, ReReduce) ->
  [H|T] = Values,
  lists:foldl(fun(Req, LatestReq) -> {[_, {<<"timestamp">>,LatestTimestamp}]} = LatestReq,
                                     {[_, {<<"timestamp">>,Timestamp}]} = Req, 
                                     if LatestTimestamp < Timestamp -> Req; true -> LatestReq end
              end, H, T)
end.
